contracts/verifications/V00_UserRegistry.sol

```diff
@@ -47,22 +50,26 @@ contract V00_UserRegistry is Owned {
         public
     {
         users[msg.sender] = _identity;
+        owners[_identity] = msg.sender;
```

Comment:
I think we should a check to this one, that verifies that `msg.sender` is in control of given identity. This can be done similar to the [DidRegistry](https://github.com/evannetwork/smart-contracts-core/blob/v2.8.0/contracts/did/DidRegistry.sol#L41).

Checking control of identity would prevent overwriting another accounts identity ownership, e.g.
- account 1 registers identity a --> a's owned by account 1
- account 2 also registers for  identity a --> a's owner is now account 2